frontend/components/customizer/path-builder.tsx

```diff
@@ -106,10 +129,14 @@ export default function PathBuilder({
         // objects.
         tokens
           .filter(t => typeof t !== 'string')
-          .reduce((accum, { name }) => {
-            accum[name] = ''
-            return accum
-          }, {})
+          .map(t => t as Key)
```

Comment:
I'm not familiar with `path-to-regexp`, but I take it that any strings in the `tokens` array are still removed/handled?

Reply:
Yea, they’re being filtered out just above.

Reply:
🤦‍♂ 

Sorry about that! for some reason my brain read that (line 108/131) as a removed line